src/components/editor/inputs/LookupTabs.jsx

```diff
@@ -48,44 +48,55 @@ const LookupTabs = (props) => {
     )
   }
 
+  const handleTabClick = (event, authorityUri) => {
+    setCurrentAuthorityUri(authorityUri)
+    event.preventDefault()
+  }
+
+  /* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
does this eslint rule need to be re-enabled?

Reply:
Fixing.